webpack.config.js

```diff
@@ -31,6 +36,17 @@ module.exports = {
           },
         },
       },
+      {
+        test: /\.s[ac]ss$/i,
+        use: [
+          // fallback to style-loader in development
+          process.env.NODE_ENV !== 'production'
+            ? 'style-loader'
+            : MiniCssExtractPlugin.loader,
+          'css-loader',
+          'sass-loader',
+        ],
```

Comment:
Doesn't seems like the CSS is minified? http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/tees-CIRCLE-38569-webpack-styles-preview/assets/app-2b4a22891844d95d2762af302515cbd0.css

Reply:
Added webpack minification.